### DISCUSSION TITLE
Failed to upload to Apple store

### DISCUSSION THREAD

**Discussion by Vietdq-dev** [codemagic-ci-cd/codemagic-docs#1944] [Q&A]
### If you are looking for help, please confirm the following...

- [X] I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
- [X] I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- [X] I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

I have published the app to the apple store, but we getting an error

### Expected results

Upload to apple store success

### Actual results

I get error same image 
<img width="992" alt="Screenshot 2023-08-04 at 17 51 34" src="https://github.com/codemagic-ci-cd/codemagic-docs/assets/80184186/bade8dfa-c72f-44f6-86da-b17730f4fe07">

### Build id (optional)

64ccd5fcf1a61d60fc42a200

**Comment by dtrdic** [comment]
Hello @Vietdq-dev, it seems to me that you might be missing a few steps before you can create and publish a .ipa file. 
Kindly take a look at this [guide](https://docs.codemagic.io/yaml-quick-start/building-a-flutter-app/) for a step-by-step process to follow. 

Additionally, you can refer to this [example](https://github.com/codemagic-ci-cd/codemagic-sample-projects/blob/main/flutter/flutter-android-and-ios-yaml-demo-project/codemagic.yaml#L51) workflow

**Reply by Vietdq-dev** [reply]
@dtrdic but I build with other app (same source), it work 64cce84e017dea613c62610b

**Reply by Vietdq-dev** [reply]
@dtrdic Could you check it?

**Comment by Vietdq-dev** [comment]
Please close it @dtrdic
